kubectl-cloudflow/cloudflowapplication/cloudflowapplication.go

```diff
@@ -46,7 +46,7 @@ type Endpoint struct {
 // Deployment contains a streamlet deployment
 type Deployment struct {
 	ClassName     string                  `json:"class_name"`
-	Config        json.RawMessage         `json:"config"`
+	Config        json.RawMessage         `json:"config,omitempty"`
```

Comment:
Did you test this? I just remember that we did an emty config `{}` instead, I'm fine with it, just vaguely remember an issue with this, maybe on the scala side..

Reply:
It's also possible that we always fill it in on the golang side, but then it is better not to add omitempty here. (also not sure why this change is needed, it changes the contract, but I think not necessary in this change?)

Reply:
I tested it and it works - will check again ..